examples/jsm/exporters/GLTFExporter.js

```diff
@@ -1185,8 +1185,9 @@ class GLTFWriter {
 
 		if ( material.emissive ) {
 
-			// note: `emissive` is not scaled by `material.emissiveIntensity` for now to accommodate glTF spec. see #21849.
-			const emissive = material.emissive.toArray();
+			// note: `emissiveIntensity` is clamped between 0 and 1 to accommodate glTF spec. see #21849 and #22000.
+			const emissiveIntensity = Math.min( Math.max( material.emissiveIntensity, 0 ), 1 );
+			const emissive = material.emissive.clone().multiplyScalar( emissiveIntensity ).toArray();
```

Comment:
@WestLangley but if we do that, then we might end up with a color that is much different than the original, right ? (in most cases, just white)

Reply:
Ok scrap my previous comment, I see what you mean. If one component exceeds 1, then scale ALL components by (1 / max component). It will indeed keep the right hue and scale closer to the intended `emissiveIntensity` than if we just clamp it before scaling

Reply:
Right. Something like this:
```js
const emissive = material.emissive.clone().multiplyScalar( material.emissiveIntensity );

const maxEmissiveComponent = Math.max( emissive.r, emissive.g, emissive.b );

if ( maxEmissiveComponent > 1 ) {

	emissive.multiplyScalar( 1 / maxEmissiveComponent );

	console.warn( 'THREE.GLTFExporter: Some emissive components exceed 1; emissive has been limited' );

}

if ( maxEmissiveComponent > 0 ) {

	materialDef.emissiveFactor = emissive.toArray();

}
```

Reply:
@SBRK I think @WestLangley is more readable 👍

Reply:
You're right. I implemented his suggestion